### PR TITLE
LTE test harness: Bug fix to check that hardware is connected before …

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**
…running tests

Addresses #68. Since device Tx and Rx classes are parameterized, CheckForHardware() cannot go into TestClassSetup block since, Tx and Rx classes are not known at test class setup time. Instead, running the test method, CheckForHardware()  externally is the option. 

Signed-off-by: Srikanth Pagadarai <srikanth.pagadarai@analog.com>

### CODE REVIEW

File: test/LTE_Tests/AD936x_LTETests.m [hunk 1]
```diff
@@ -76,6 +76,16 @@ function ConfigHW(testCase, TxClass, RxClass)
     end
     
     methods(Test)
+        function CheckForHardware(testCase, AD936xClassTx, ...
```
Comment (tfcollins) [inline test/LTE_Tests/AD936x_LTETests.m:79]: Why not do this at within the test setup methods?

File: test/LTE_Tests/runLTETests.m [hunk 2]
```diff
@@ -54,10 +54,19 @@
...
             error('%s unsupported for LTE test harness', BoardName);
     end
-    params = Parameter.fromData('AD936xClassTx',AD936xClassTx, ...
-        'AD936xClassRx',AD936xClassRx, ...
+    % check that hardware is connected
+    params = Parameter.fromData('AD936xClassTx', AD936xClassTx, ...
+        'AD936xClassRx', AD936xClassRx);
+    suite = TestSuite.fromMethod(?AD936x_LTETests, 'CheckForHardware', ...
+        'ExternalParameters', params);
+    run(suite);
```
Comment (tfcollins) [inline test/LTE_Tests/runLTETests.m:62]: If the test fails what happens? Does it just keep running?

File: test/LTE_Tests/AD936x_LTETests.m (hunk 1 above)
Comment (SrikanthPagadarai) [reply test/LTE_Tests/AD936x_LTETests.m]: Addressed by the latest commit. ClassSetupParameter properties block helps overcome issues I was facing earlier with using TestClassSetup  methods block.

File: test/LTE_Tests/runLTETests.m (hunk 2 above)
Comment (SrikanthPagadarai) [reply test/LTE_Tests/runLTETests.m]: Addressed by the latest commit. Test is labeled incomplete.

**Pull request merged**